src/Moontoast/Math/BigNumber.php

```diff
@@ -647,10 +647,11 @@ public static function convertFromBase10($number, $toBase)
         $outNumber = '';
         
         while (bccomp($number, $toBase) >= 0) {
-            $quotient = bcdiv($number, $toBase, 0);
-            $remainder = bcsub($number, bcmul($quotient, $toBase, 0), 0);
+            list($number, $remainder) = array(
+                bcdiv($number, $toBase, 0),
+                bcmod($number, $toBase)
+            );
```

Comment:
Nevermind. I see what would happen if you did this this way. `$number` is overwritten by the first operation, and then `bcmod()` uses the new number, whereas, using `list`, you use the original `$number` in both first before reassigning `$number`.